src/components/charts/BarChart.tsx

```diff
@@ -146,7 +147,9 @@ const BarChart = () => {
 
   const getSessionData = async () => {
     const params = {
-      period: DEFAULT_PERIOD,
+      period: location.pathname.includes("home")
+        ? DURATION30MIN
```

Comment:
Not `DURATION30MIN` what if we decide to change the duration, then we have to rename the variable.
Change it to `HOME_PAGE_REFERSH_DURATION`